src/backend/distributed/commands/utility_hook.c

```diff
@@ -202,7 +202,8 @@ multi_ProcessUtility(PlannedStmt *pstmt,
 		ErrorIfUnstableCreateOrAlterExtensionStmt(parsetree);
 	}
 
-	if (IsA(parsetree, CreateExtensionStmt))
+	if (IsA(parsetree, CreateExtensionStmt) && !CitusHasBeenLoaded()
```

Comment:
Is this not blocking _all_ extension create statements? Having said that, I'm not sure what other extensions can/might be created before Citus is loaded.

Reply:
I don't believe this will block all extension create statements because it's a separate conditional statement from the conditional statement below it that only checks '!CitusHasBeenLoaded()' and will call the 'PrevProcessUtilityHook()' if that is true.